src/book.py

```diff
@@ -569,13 +574,11 @@ def _read_kraken_ledgers(self, file_path: Path) -> None:
                         "Fee", utc_time, platform, fee, coin, row, file_path
                     )
 
-        assert dup_state["deposit"] == 0, (
-            "Orphaned deposit. (Must always come in pairs). " "Is your file corrupted?"
-        )
-        assert dup_state["withdrawal"] == 0, (
-            "Orphaned withdrawal. (Must always come in pairs). "
-            "Is your file corrupted?"
-        )
+        if margin_warnings:
+            log.warning(
+                f"{file_path}: {margin_warnings} margin entries. Margin trading is "
+                "currently not supported. Please create an Issue or PR."
+            )
```

Comment:
Shouldn't this be an error as this fucks up the entire tax calculation? Or is margin trading completly separated from normal trading?

Reply:
Good point, I think this depends if the margin positions are closed or settled, and if closed, what the base for the trade was. If the gain/loss was payed in fiat/EUR at the end of the margin trade, then it should be fine. I need to think about it.
This leads to the question: Are gains/losses in fiat currency actually taxable? Let's say I trade BTC/EUR, then BTC/USD, then BTC/EUR, is the change in USD/EUR between the trades also relevant for taxation?

Reply:
To answer my own question: Yes, gains/losses in other fiat currencies are taxable (refer to e.g. https://www.iww.de/astw/archiv/--23-estg--fremdwaehrungsgeschaefte-fuehren-zur-steuerpflicht-f51260)

Reply:
To be on the safe side, I think it's best to tackle margin trading together with #52 and throw an error for now.